pkg/util/tls/tls.go

```diff
@@ -32,15 +30,13 @@ func (cfg *ClientConfig) GetTLSConfig() (*tls.Config, error) {
 	if cfg.CertPath != "" && cfg.KeyPath != "" && cfg.CAPath != "" {
 		clientCert, err := tls.LoadX509KeyPair(cfg.CertPath, cfg.KeyPath)
 		if err != nil {
-			level.Error(util.Logger).Log("msg", "error loading certs", "error", err)
-			return nil, err
+			return nil, fmt.Errorf("failed to load TLS certs: %v", err)
 		}
 
 		var caCertPool *x509.CertPool
 		caCert, err := ioutil.ReadFile(cfg.CAPath)
 		if err != nil {
-			level.Error(util.Logger).Log("msg", "error loading ca cert", "error", err)
-			return nil, err
+			return nil, fmt.Errorf("error loading ca cert: %v", err)
```

Comment:
Please use `errors.Wrapf()` (from `github.com/pkg/errors`) and also add the path of the files (following my previous comment).